rust/core-lib/src/editor.rs

```diff
@@ -1058,6 +1058,17 @@ impl<W: Write + Send + 'static> Editor<W> {
         Some(text.slice_to_string(offset, end_off))
     }
 
+    pub fn plugin_get_selections(&self, view_id: &ViewIdentifier) -> Value {
+        //TODO: multiview support
```

Comment:
I've always been uneasy about the whole separation between the active view and the non-active views into two separate containers, and further the fact that RPCs for the views have to dispatch through the `Editor`, and this just makes me more uneasy.

I don't know how to fix this other than a large refactor so for now this is just a general comment about a sense of foreboding.